public/stylesheets/game.css

```
@@ -0,0 +1,127 @@
+body {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  height: 100dvh;
+  background-image: url(/images/background-image.png);
+  background-size: cover;
+  background-position: center;
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+[data-screen="game"] {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem;
+}
+
+.game-header {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+}
+
+.game-header .point {
+  font-size: 75px;
+  background: #ced;
+  border: 1px solid rgb(1, 64, 100);
+  border-radius: 30px;
+  width: 720px;
+  height: 120px;
+  line-height: 120px;
+  color: #000000;
+  text-align: center;
+}
+
+.game-header .timer {
+  font-size: 75px;
+  font-weight: bold;
+  background: #ced;
+  border: 1px solid rgb(1, 64, 100);
+  border-radius: 30px;
+  width: 360px;
+  height: 120px;
+  line-height: 120px;
+  color: #000000;
+  text-align: center;
+}
+
+.game-body {
+  width: 100%;
+  display: flex;
+  flex-grow: 1;
+}
+
+.game-body .typing {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
```

`align-items: center;`追加
```suggestion
  justify-content: space-around;
  align-items: center;
```